UserDetailsClient/UserDetailsClient.Droid/UserDetailsClient.Droid.csproj

```diff
@@ -35,6 +35,7 @@
     <WarningLevel>4</WarningLevel>
     <AndroidUseSharedRuntime>True</AndroidUseSharedRuntime>
     <AndroidLinkMode>None</AndroidLinkMode>
+    <EmbedAssembliesIntoApk>true</EmbedAssembliesIntoApk>
```

Comment:
This is needed in order for the the app to deploy to an android 11 device. Probably a bug somewhere in Xamarin.
I think this is what you ran into last week @jmprieur